src/Func.cc

```diff
@@ -280,10 +295,10 @@ void Func::CheckPluginResult(bool handled, const ValPtr& hook_result, FunctionFl
 namespace detail
 	{
 
-ScriptFunc::ScriptFunc(const IDPtr& arg_id) : Func(SCRIPT_FUNC)
+ScriptFunc::ScriptFunc(const IDPtr& id) : Func(SCRIPT_FUNC)
```

Comment:
This is now arg_id->id renaming - I'll see if that can be removed to nothing during the merge.